website/content/docs/agent/telemetry.mdx

```diff
@@ -755,3 +755,28 @@ Consul attaches the following labels to metric values.
 | `peer_id`                             | The ID of a peer connected to the reporting cluster or leader.                   | Any UUID                                  |
 | `partition`                           | <EnterpriseAlert inline /> Name of the partition that the peering is created in. | Any defined partition name in the cluster |
 
+## Server Host Metrics
+
+Consul servers report the following metrics about the host's system resources
```

Comment:
We should add a note about how this currently only works on VMs, so there are no surprises to users if they attempt to collect metrics inside a container.